packages/@ama-sdk/schematics/schematics/typescript/core/openapi-codegen-typescript/src/main/java/com/amadeus/codegen/ts/LambdaHelper.java

```diff
@@ -426,9 +426,9 @@ public RemoveUnusedImports() {
         @Override public String formatFragment(String fragment) {
             //Retrieve all the imported classes
             List<String> allMatches = new ArrayList<String>();
-            Matcher importLinesMatcher = Pattern.compile("import\\s\\{(.*)}.*;").matcher(fragment);
+            Matcher importLinesMatcher = Pattern.compile("import\\s+\\{(.*)}.*;").matcher(fragment);
```

Comment:
You do not need to support the `import type ....` here? (same for the other `import\\s+` patterns)
```suggestion
            Matcher importLinesMatcher = Pattern.compile("import\\s+(?:type\\s+)?\\{(.*)}.*;").matcher(fragment);
```